www/src/pages/editor/EditorPage.tsx

```diff
@@ -1,15 +1,38 @@
-import { Flex } from "antd";
+import { Button, Flex } from "antd";
 import { observer } from "mobx-react-lite";
 import { SearchBar, useSearchArgs } from "./SearchBar";
 import { DedupMineralSiteTable } from "./DedupMineralSiteTable";
+import { useRef } from "react";
+import { NewMineralSiteModal, newMineralSiteFormRef } from "./NewMineralSiteModal";
 
 export const EditorPage = observer(() => {
   const [searchArgs, normSearchArgs, setSearchArgs] = useSearchArgs();
-
+  const modalRef = useRef<newMineralSiteFormRef>(null);
```

Comment:
I think you change the wrong name, I mean change `modalRef` to `newMineralSiteFormRef`